packages/bpk-component-rating/README.md

```diff
@@ -12,40 +12,47 @@ npm install bpk-component-rating --save-dev
 
 ```js
 import React from 'react';
-import BpkRating, { RATING_SIZES, RATING_SCALES, RATING_TYPES } from 'bpk-component-rating';
+import BpkRating, { RATING_SIZES, RATING_SCALES } from 'bpk-component-rating';
 
 export default () => (
   <BpkRating
-    ariaLabel="9 Excellent would recommend"
+    ariaLabel="4.6 Excellent 2,420 reviews"
+    value={4.6}
     title="Excellent"
-    subtitle="It was almost perfect!"
-    value={9}
+    subtitle="2,420 reviews"
   />
 
-  // Large rating
+  // Large size rating
   <BpkRating
-    ariaLabel="9 Excellent would recommend"
+    ariaLabel="4.5 Excellent would recommend"
```

Comment:
Don't always use "Excellent" rating as the [example](.storybook/__image_snapshots__/storyshots-test-js-visual-tests-bpk-component-rating-visual-test-1-snap.png), I think we should include "Average", "Bad" even there are used the same design

Reply:
Sure thing, have update example and some test cases